pkg/domain/infra/abi/images.go

```diff
@@ -121,12 +121,11 @@ func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entiti
 	}
 
 	var registryCreds *types.DockerAuthConfig
-	if options.Credentials != "" {
-		creds, err := util.ParseRegistryCreds(options.Credentials)
-		if err != nil {
-			return nil, err
+	if len(options.Username) > 0 && len(options.Password) > 0 {
```

Comment:
It is not possible to have a username without password?

Reply:
Only looking for username now, thanks!